ansible/playbooks/AdoptOpenJDK_Unix_Playbook/group_vars/all/adoptopenjdk_variables.yml

```diff
@@ -11,29 +11,29 @@ Sudoers_File: /etc/sudoers
 
 # Jenkins User Variables:
 Jenkins_Username: jenkins
-Jenkins_User_SSHKey: /Vendor_Files/keys/id_rsa.pub
+Jenkins_User_SSHKey: ~/Vendor_Files/keys/id_rsa.pub
```

Comment:
Not sure why you've modified this. It has nothing to do with this new machine and will result in a change to how others who are running the playbooks have to store the Vendor_Files stuff.